src/press/models.py

```diff
@@ -312,6 +314,22 @@ def preprint_dois_enabled(self):
     def code(self):
         return 'press'
 
+    @property
+    def active_journals(self):
+        from journal import models as journal_models
+        # imported here to avoid circular imports
+        return journal_models.Journal.objects.filter(
+            hide_from_press=False,
+        )
+
+    @property
+    def published_articles(self):
+        return submission_models.Article.objects.filter(
+            journal__press=self,
```

Comment:
`Journal` does not have an FK to `Press` so you can't filter on it here.

Reply:
Thanks for catching that. Just removed it.